ci-operator/config/openshift-kni/commatrix/openshift-kni-commatrix-main__nightly-4.16.yaml

```diff
@@ -0,0 +1,79 @@
+build_root:
+  from_repository: true
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.16"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+  ppc64le-latest:
+    candidate:
+      architecture: ppc64le
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: commatrix-e2e-tests
+  cron: '@weekly'
+  steps:
+    cluster_profile: aws-telco
+    env:
+      BASE_DOMAIN: telco5g-ci.devcluster.openshift.com
+      USER_TAGS: |
+        ci-job-source prowci
+        ci-job-branch master
+        ci-job-fullname openshift-kni-commatrix-e2e-tests-master
+        ci-job-type commatrix-e2e-tests
+        ci-repo-name commatrix
+        ci-org-name openshift-kni
+    test:
+    - as: commatrix-e2e-tests
+      commands: go mod vendor && make e2e-test
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - ref: openshift-e2e-test
```

Comment:
You can remove this as well, the e2e-test is included in the workflow https://steps.ci.openshift.org/workflow/openshift-e2e-aws-ovn-serial